### PR TITLE
maintain tree hierarchy when filtering

### PR DESCRIPTION
**Pull request by shakeh**

### CODE REVIEW

**Comment by dlamoris** [comment]
This pull request **introduces 1 alert** when merging b5229b94a1026f2d387c05f4b56d5d698487a4b1 into 07dad645211cb08df99ebc88b954dbb80eff02ec - [view on LGTM.com](https://lgtm.com/projects/g/Open-MBEE/ve/rev/pr-9eda4028384a88ecdb1552ed0b45e70d9f035ea0)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by dlamoris** [comment]
This pull request **fixes 1 alert** when merging af9f621f1b9b7b0044fc3e00458ffbff96246e8e into 07dad645211cb08df99ebc88b954dbb80eff02ec - [view on LGTM.com](https://lgtm.com/projects/g/Open-MBEE/ve/rev/pr-4ac938bd077f17e7f9da555131a0a906e1e20551)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/mms.directives.js [hunk 1]
```diff
@@ -71,12 +71,14 @@
...
  */
-angular.module('mms.directives', ['mms', 'mms.directives.tpls', 'ui.bootstrap', 'angular-growl', 'angularjs-dropdown-multiselect'])
-.config(['$sceProvider', 'growlProvider', function($sceProvider, growlProvider) {
+angular.module('mms.directives', ['mms', 'mms.directives.tpls', 'ui.bootstrap', 'angular-growl', 'angularjs-dropdown-multiselect', 'ui.tree-filter'])
+.config(['$sceProvider', 'growlProvider', 'uiTreeFilterSettingsProvider', function($sceProvider, growlProvider, uiTreeFilterSettingsProvider) {
     $sceProvider.enabled(false);
     growlProvider.onlyUniqueMessages(false);
     growlProvider.globalTimeToLive({success: 5000, error: -1, warning: 5000, info: 5000});
     growlProvider.globalPosition('bottom-right');
+    uiTreeFilterSettingsProvider.addresses = ['label'];
```
Comment (dlamoris) [inline src/mms.directives.js]: can use branch.data.name instead of label, since that's what we actually show in the tree

Comment (shakeh) [reply src/mms.directives.js]: for some reason the presentation elements stop working with the filter when i use 'branch.data.name' going to leave label for now

**Comment by dlamoris** [comment]
This pull request **fixes 1 alert** when merging 5ee8955db8cb65c4c0480d6819aab9966f62b032 into 3764bf53a22ff7bfd9604b5050043998a1591da5 - [view on LGTM.com](https://lgtm.com/projects/g/Open-MBEE/ve/rev/pr-cdc97f9274dbe9ce1616ed84456f014f5077761d)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**